modules/openapi-generator/src/main/resources/JavaJaxRS/pojo.mustache

```diff
@@ -1,6 +1,13 @@
 /**
  * {{#description}}{{.}}{{/description}}{{^description}}{{classname}}{{/description}}
  */{{#description}}
+{{#jackson}}
+@JsonPropertyOrder({
+{{#vars}}
+  {{classname}}.JSON_PROPERTY_{{nameInSnakeCase}},
```

Comment:
Maybe don't output the `,'  for the last var ?

Reply:
It compiles, but is is true that it is not nice. Thank you for the input!